app/models/sponsorship.rb

```diff
@@ -1,6 +1,6 @@
 class Sponsorship < ApplicationRecord
   LEVELS = %w[gold silver bronze tag media devrel].freeze
-  LEVELS_WITH_EXPIRATION = %w[gold silver bronze].freeze
+  METAL_LEVELS = %w[gold silver bronze].freeze
```

Comment:
ahahha I laughed at `METAL_LEVELS` :D love it!

Reply:
Naming problems :D